com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -235,6 +235,27 @@ public void UnregisterSideChannel(SideChannel channel)
             Communicator?.UnregisterSideChannel(channel);
         }
 
+        /// <summary>
+        /// Returns the SideChannel of Type T if there is one registered, or null if it doesn't.
+        /// If there are multiple SideChannels of the same type registered, only the first found will be returned.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
+        public T GetSideChannel<T>() where T: SideChannel
```

Comment:
@vincentpierre @surfnerd I think we had talked about this before. It seems like a clean way to get access to sidechannels without adding a custom method to Academy for each one.

Reply:
I can replace Academy.FloatProperties with this later if we're happy with it.

Reply:
Let's replace Academy.Instance.FloatProperty then. Do you want to do it in this PR or make another one?

Reply:
Let's do it in another PR.

Reply:
https://jira.unity3d.com/browse/MLA-799